src/sagas/login.js

```diff
@@ -230,8 +229,8 @@ function* initialLoad () {
   // requires storage domains to be in redux store
   yield put(getIsoFiles())
 
-  // --- store is populated with user & system info, now pull VMs and Pools
-  yield put(getByPage({ page: 1 }))
+  // The `Vms` card view component will take care of loading pages of VMs and Pools as needed.
```

Comment:
We already merged this change in another PR right?

Reply:
Fixed with rebase.